release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -369,6 +369,19 @@ This release adds improvements related to the following components and concepts.
 
 Pods created using the macvlan CNI plugin, where the IP address management CNI plug-in has assigned IPs, now send IPv6 unsolicited neighbor advertisements by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh IPv6 neighbor caches.
 
+[id="ocp-4-15-egressfirewall-updates-status-management"]
+==== Status management updates for EgressFirewall and AdminPolicyBasedExternalRoute CR
+
+The following updates have been made to the status management of `EgressFirewall` and `AdminPolicyBasedExternalRoute` policy:
```

Comment:
```suggestion
The following updates have been made to the status management of `EgressFirewall` and `AdminPolicyBasedExternalRoute` custom-resource policy:
```
CR is the headline, so not sure if it should be in the note itself

Reply:
I added this in, but it should be [custom](https://redhat-documentation.github.io/supplementary-style-guide/#custom-resource) resource